src/compiler/compiler.lisp

```diff
@@ -1113,27 +1113,28 @@
         (throw "Division by zero"))
     (return (% ,x ,y))))
 
+;;; @VKM path 30.11
+(define-builtin %ash-left (x y)
+  `(call-internal |Bitwise_ash_L| ,x ,y))
```

Comment:
Do we need the auxiliary functions here?  Probably now, right?  we could use `(<< ,x ,y)` ?

```
(js '(<< x 2)) ; => x<<2;
```

Reply:
Its workaround https://github.com/jscl-project/jscl/issues/375#issue-750906096
While there is no certainty that compilation priorities always work correctly, so be it
I would not like to puzzle now why "this does not work"